diff --git a/node_modules/react-native/index.js b/node_modules/react-native/index.js
index 2c47d5f..f9f1660 100644
--- a/node_modules/react-native/index.js
+++ b/node_modules/react-native/index.js
@@ -358,7 +358,7 @@ module.exports = {
       .VirtualViewMode;
   },
   // #endregion
-} as ReactNativePublicAPI;
+};
 
 if (__DEV__) {
   /* $FlowFixMe[prop-missing] This is intentional: Flow will error when
